app/views/articles/index.html.erb

```diff
@@ -95,7 +95,7 @@
               </h3>
             </div>
             <a href="/<%= @featured_story.cached_user.username %>" class="featured-profile-button">
-              <img class="featured-profile-pic" src="<%= ProfileImage.new(@featured_story.cached_user).get(width: 90) %>" alt="<%= @featured_story.title %>" />
+              <img class="featured-profile-pic" src="<%= ProfileImage.new(@featured_story.cached_user).get(width: 90) %>" width="40" height="40" alt="<%= @featured_story.title %>" />
```

Comment:
I was wondering, but maybe you chose this because the height would be too short, if the width of the <img tag is 40, why not also ask for a width: 40 in the method? Could you try and see what happens?

`ProfileImage.new(story.cached_user).get(width: 40)`

Reply:
I don't know what the ideal answer is, but if we're already fetching `width: 90` elsewhere, it means that image will be cached in more places, so it's more efficient to keep fetching that.

Width 90 is sort of our standard small profile size.